### PR TITLE
adding verbose

### PR DESCRIPTION
**Pull request by raphaelsaavedra**

### CODE REVIEW

File: src/estimation.jl [hunk 1]
```diff
@@ -53,7 +53,8 @@ end
 
 Estimate structural model hyperparameters
 """
-function estimate_statespace(model::StateSpaceModel, nseeds::Int; f_tol = 1e-10, g_tol = 1e-10, iterations = 10^5)
+function estimate_statespace(model::StateSpaceModel, nseeds::Int; f_tol = 1e-10, g_tol = 1e-10,
```
Comment (guilhermebodin) [inline src/estimation.jl]: ` f_tol::Float64 = 1e-10, g_tol::Float64 = 1e-10, iterations::Int = 10^5`

**Comment by guilhermebodin** [comment]
I think there are some infos in the constructors, we could just take them out

File: src/estimation.jl (hunk 1 above)
Comment (raphaelsaavedra) [reply src/estimation.jl]: Ok

**Comment by raphaelsaavedra** [comment]
> I think there are some infos in the constructors, we could just take them out

True

**Pull request merged**